app/models/changeset/pull_request.rb

```diff
@@ -131,8 +131,12 @@ def risky?
   def risks
     return @risks if defined?(@risks)
     @risks = parse_risks(@data.body.to_s)
-    @missing_risks = @risks.nil?
-    @risks = nil if @risks&.match?(/\A\s*\-?\s*None\Z/i)
+    if @risks&.match?(/\A\s*\-?\s*None\Z/i)
+      @risks = nil
```

Comment:
It looks like the only job of `@risks` is to be asked the question `risks.present?` by `#risky?`. Would it be better to encapsulate this logic in a small object?

```rb
class Risk
  def initialize(string)
    # ...
  end

  def risky?
    # ...
  end
end
```

Reply:
possible ... there are lots of ways to refactoring this, but that can be it's own PR, trying to keep this simple :)